portfolio/src/main/webapp/style.css

```diff
@@ -15,3 +15,23 @@ img{
     width: 40%;
    
 }
+
+div.gallery{
+    margin:5px;
+    border:1px solid #ccc;
+    float: left;
+    width:auto;
```

Comment:
make the indentation of these properties be 2 spaces. For example,

div.gallery {
  margin: 5px; (add a space after the : here)
  border: 1px solid #ccc;
  ...
}

Same applies to the others in this file!